plugins/com.google.cloud.tools.eclipse.appengine.login/src/com/google/cloud/tools/eclipse/appengine/login/ui/AccountSelector.java

```diff
@@ -100,9 +99,26 @@ public int selectAccount(String email) {
     return index;
   }
 
+  /**
+   * Works the same way as {@code selectAccount} does, except that if there exists only one
+   * account signed in, that account will always be selected regardless of {@code email}.
+   *
+   * @see #selectAccount
+   */
+  public int selectAccountInSingleAccountSelectMode(String email) {
```

Comment:
we really need an extra method here? N way to include this logic in selectAccount?